src/data/polynomial/degree/definitions.lean

```diff
@@ -999,6 +1006,12 @@ by rw [X_pow_eq_monomial, degree_monomial _ (@one_ne_zero R _ _)]
 @[simp] lemma nat_degree_X_pow (n : ℕ) : nat_degree ((X : R[X]) ^ n) = n :=
 nat_degree_eq_of_degree_eq_some (degree_X_pow n)
 
+lemma nat_degree_X_pow_le (n : ℕ) : (X ^ n : R[X]).nat_degree ≤ n :=
+begin
+  nontriviality R,
```

Comment:
`nontrivial R` is already in the context here, no?

Reply:
Robert, thank you very much for catching this!  I *really* wanted to have the statement without assuming `nontrivial`, so I've updated the PR!